src/components/svgs/stations/qingdao-metro-basic.tsx

```diff
@@ -0,0 +1,238 @@
+import { RmgFields, RmgFieldsField } from '@railmapgen/rmg-components';
```

Comment:
If we only have one station, how about calling it QingdaoMetroStation instead of something basic?